frontend/src/lib/components/ArgInput.svelte

```diff
@@ -233,6 +233,12 @@
 	let itemsLimit = 50
 
 	$: validateInput(pattern, value, required)
+
+	let oldValue = value
+	$: if (JSON.stringify(oldValue) !== JSON.stringify(value)) {
```

Comment:
deepEqual, also no inlined reactivity please, hard to read

Reply:
also had a prop that this should fire events on change, most ArgInput's dont need that kind of reactivity and ArgInput is used everywhere

Reply:
Also add debounce here, no need to fire events on every key stroke